### PR TITLE
Create 2018-12-20-coreweekly-week-48-2018.md

### PR DESCRIPTION
**Pull request by ttoine**

### CODE REVIEW

File: news/_posts/core-weekly/2018-12-20-coreweekly-week-48-2018.md [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+This edition of the Core Weekly report highlights changes in PrestaShop's core codebase during the last week, from Monday 03rd to Sunday 09th of December 2018.
+
+![Core Weekly banner](/assets/images/2018/12/banner-core-weekly.jpg)
+
+
+## General messages
+
+Dear developers,
+
+We are one day before the week-end and the Holiday period. Jingle bells, Jingle bells :-)
```
Comment (eternoendless) [inline news/_posts/core-weekly/2018-12-20-coreweekly-week-48-2018.md]: Holiday season?

File: news/_posts/core-weekly/2018-12-20-coreweekly-week-48-2018.md [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+
+At PrestaShop, we would like to thank the most important contributors of the year, and we have already some goodies in stock. What do you think? What king of thing would you like and be proud to wear or to use? Please comment this article or get in touch with us.
```
Comment (eternoendless) [inline news/_posts/core-weekly/2018-12-20-coreweekly-week-48-2018.md]:
```suggestion
 At PrestaShop, we would like to thank the most important contributors of the year, and we have already some goodies in stock. What do you think? What kind of things would you like and be proud to wear or to use? Please comment this article or get in touch with us.
```

File: news/_posts/core-weekly/2018-12-20-coreweekly-week-48-2018.md (hunk 1 above)
Comment (ttoine) [reply news/_posts/core-weekly/2018-12-20-coreweekly-week-48-2018.md]: done

Comment (eternoendless) [inline news/_posts/core-weekly/2018-12-20-coreweekly-week-48-2018.md]:
```suggestion
We are one day before the week-end and the Holidays season. Jingle bells, Jingle bells :-)
```

**Pull request merged**